### PR TITLE
Disburse maturity destination input

### PR DESCRIPTION
**Pull request by mstrasinskis**
# Motivation

Add destination address support (any icrc address) to disburse maturity modal and minor design/copy changes.

# Changes

- add input for the destination
- add `toAccount` field to disburseMaturity service
- add `NeuronSelectMaturityDisbursement` component to combine maturity disbursement configuration

# Tests

- NeuronSelectMaturityDisbursement.page-object.ts
- Updated SnsDisburseMaturity.spec to support updated structure and destination data

# Todos

- [ ] Add entry to changelog (if necessary).
Not necessary.

# Screenshots

image

image

image

### CODE REVIEW

**Review by lmuntaner — COMMENTED** [review]
Shouldn't we default to the Main account?

And allow selection of user accounts with a dropdown?

The same way we have for all destinations.

We have a component for it: `SelectDestinationAddress.svelte`

File: frontend/src/lib/components/neuron-detail/NeuronSelectMaturityDisbursement.svelte [hunk 1]
```diff
@@ -0,0 +1,92 @@
+<script lang="ts">
```
Comment (mstrasinskis) [inline frontend/src/lib/components/neuron-detail/NeuronSelectMaturityDisbursement.svelte:1]: A component just to isolate the inputs logic

File: frontend/src/lib/modals/neurons/DisburseMaturityModal.svelte [hunk 2]
```diff
@@ -51,39 +104,87 @@
...
+<style lang="scss">
+  .content {
+    margin-top: var(--padding-3x);
+    display: flex;
+    flex-direction: column;
+    gap: var(--padding-2x);
+  }
+
+  .destination-key {
+    white-space: nowrap;
```
Comment (mstrasinskis) [inline frontend/src/lib/modals/neurons/DisburseMaturityModal.svelte:184]: A request to not break "To address:" line.

File: frontend/src/lib/modals/neurons/DisburseMaturityModal.svelte [hunk 3]
```diff
@@ -51,39 +104,87 @@
...
+  }
+
+  .destination-value {
+    text-align: right;
```
Comment (mstrasinskis) [inline frontend/src/lib/modals/neurons/DisburseMaturityModal.svelte:188]: The value according to design needs to be right aligned. It feels that KeyValue component needs a review w/ designer.

**Comment by mstrasinskis** [comment]
Obsolete.

**Pull request closed without merging**